.gitignore

```diff
@@ -0,0 +1,3 @@
+composer.lock
```

Comment:
I think we should commit the composer.lock to install only the right versions of the packages.

Reply:
nope, composer.lock only works for projects, not for dependencies/packages.

Reply:
I agree with @dpfaffenbauer, to narrow down the packages we could use in the require section a more specific version.